lib/git.js

```diff
@@ -33,8 +33,14 @@ exports.getCommits = function (options) {
       revisions = tag ? tag + '..HEAD' : '';
     }
 
+    var gitLogCommand = 'git log -E --format=' + FORMAT + ' ' + revisions;
+
+    if (options.subDirectory) {
```

Comment:
It looks like if you do `git log -- .`, it behaves the same way as `git log`, so it might simplify the logic if you make the default value of `subDirectory` `.` in `lib/cli.js`.

Reply:
The combination of using default value for 'subDirectory' and discarding all the explicit path handling - make this PR way simpler 😅 
Liked it way better though 👍